packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -1368,15 +1370,15 @@ void main() {
               children: <Widget>[
                 EditableText(
                   key: key1,
-                  controller: TextEditingController(),
+                  controller: controller1,
```

Comment:
In original test it was different controller, i.e. controller3 and controller4 for here and below. We still want this scenario to be tested.